.github/workflows/test-python-37.yml

```diff
@@ -2,9 +2,9 @@ name: test-tutorials-python-37
 
 on:
   workflow_dispatch:
-#  push:
-#    branches:
-#      - 'master'
+  push:
```

Comment:
I think we should change this condition so it is run when a review request is submitted. https://github.com/logicalclocks/hopsworks-ee/blob/933829b252299a7af1c28dc2700a8ec4478d1969/.github/workflows/hopsworks-tests.yml#L3
Otherwise from what I understand this is only run when the PR is merged, and at that point it's too late as the problems should be caught in the review process.